<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlcDeviceManageUIClass</class>
 <widget class="QWidget" name="PlcDeviceManageUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlcDeviceManageUI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pb_Getsegment">
       <property name="text">
        <string>GetSegment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_3">
       <property name="maximum">
        <number>300000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="maximum">
        <number>30000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_clearsegment">
       <property name="text">
        <string>clearSegment</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="maximum">
        <number>300000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_setwidth">
       <property name="text">
        <string>setWidth</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pb_Init">
     <property name="text">
      <string>Init</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pb_Uninit">
     <property name="text">
      <string>Uninit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pb_open">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pb_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_4">
       <property name="maximum">
        <number>3000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_GetDevice">
       <property name="text">
        <string>GetDevice</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_5">
       <property name="maximum">
        <number>3000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_6">
       <property name="maximum">
        <number>30000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_SetDevice">
       <property name="text">
        <string>SetDevice</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_7">
       <property name="maximum">
        <number>30000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>增删</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>Label</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_add">
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_delete">
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PlcDeviceManageUI.qrc"/>
 </resources>
 <connections/>
</ui>
